client/src/components/Account/AccountBox.tsx

```diff
@@ -6,7 +6,7 @@ import FriendsTab from './Friends/FriendsTab';
 
 type Props = {};
 
-function AccountBox({}: Props) {
+function AccountBox() {
   const [currentTab, setCurrentTab] = useState<string | null>('Profile');
```

Comment:
nitpicky: I think it would be a good idea to create an enum for the different tabs to avoid [magic strings](https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings). You can include this enum at the top of the file since it's probably only going to be used in this file. If we need to use these values somewhere else in the future, we can move the enum to an enums dedicated file.